docs/content/staking/epoch-preparation.mdx

```diff
@@ -16,15 +16,21 @@ description: Technical Overview of the Flow Epoch Protocol
 
 # Epochs
 
-The epoch preparation protocol defines how information about the next epoch is determined and propagated to the protocol state.
+The epoch preparation protocol defines how information about the next epoch
+is determined and propagated to the protocol state.
 
 There are two primary actors in this protocol, the Epoch Smart Contracts, and the Consensus Committee:
 
 - [`Epoch Smart Contracts`](https://github.com/onflow/flow-core-contracts/blob/master/contracts/epochs) - the smart contracts that manage epochs:
-    - `FlowClusterQC` : Manages the quorum certificate generation for bootstrapping the hotstuff consensus algorithm for each collector cluster.
-    - `FlowDKG` : Manages the Distributed Key Generation that consensus nodes participate in to initialize the random beacon for each epoch.
-    - `FlowIDTableStaking` : Manages the source of truth for the identity table, and enforces rules related to staking FLOW, delegating, paying rewards, and allocating token movements between epochs.
-    - `FlowEpoch` : Ties all of the previously mentioned contracts together to manage the high level epoch lifecycle. `FlowEpoch` acts as a state machine that transitions between different epoch phases when specific conditions from the other contracts are met and triggers important operations in the other smart contracts when phase changes happen.
+    - `FlowClusterQC` : Manages the quorum certificate generation for bootstrapping
+      the hotstuff consensus algorithm for each collector cluster.
```

Comment:
Are we still referring to Hotstuff in these docs? Or just consensus... ?

Reply:
as far as I know, collector nodes are still using hotstuff for their clusters, right?